src/test/36-search.ts

```diff
@@ -63,8 +62,9 @@ describe("validating search apis", function () {
             });
 
             it("update view properties", async () => {
-                const response = await search.updateViewProperties({ links: { test: { analyzers: ['identity'], fields: { 'v': {} } } } });
-                expect(response.links).to.deep.equal({ test: { analyzers: ['identity'], fields: { 'v': {} }, includeAllFields: false, storeValues: "none", trackListPositions: false } });
+                const response = await search.updateViewProperties({ links: { [coll]: { analyzers: ['identity'], fields: { 'v': {} } } } });
+                expect(response.links).to.deep.equal({ [coll]: { analyzers: ['identity'], fields: { 'v': {} }, includeAllFields: false, storeValues: "none", trackListPositions: false } });
+                await c8Client.deleteCollection(coll);
```

Comment:
move this to `afterAll`. Also on line number 36, move `await c8Client.createCollection(coll);` to `beforeAll`. This will make sure that the tests are not dependent on each other.

Reply:
done